jenkins_pipelines/environments/manager-3.2-reference-PRV

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env groovy
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '20', daysToKeepStr: '4')),
+    disableConcurrentBuilds(),
+    parameters([
+        string(name: 'tf_file', defaultValue: 'susemanager-ci/terracumber_config/tf_files/SUSEManager-3.2-refenv-PRV.tf', description: 'Path to the tf file to be used'),
+        string(name: 'sumaform_gitrepo', defaultValue: 'https://github.com/uyuni-project/sumaform.git', description: 'Sumaform Git Repository'),
+        string(name: 'sumaform_ref', defaultValue: 'master', description: 'Sumaform Git reference (branch, tag...)'),
+        choice(name: 'sumaform_backend', choices: ['libvirt', 'aws'], description: 'Sumaform backend to be used (see https://github.com/uyuni-project/sumaform#backend-choice)'),
+        choice(name: 'terraform_bin', choices: ['/usr/bin/terraform_bin'], description: 'Terraform binary path'),
+        choice(name: 'terraform_bin_plugins', choices: ['/usr/bin'], description: 'Terraform plugins path'),
+        string(name: 'terracumber_gitrepo', defaultValue: 'https://gitlab.suse.de/juliogonzalezgil/terracumber.git', description: 'Terracumber Git Repository'),
+        string(name: 'terracumber_ref', defaultValue: 'master', description: 'Terracumber Git ref (branch, tag...)'),
+        booleanParam(name: 'terraform_bin_init', defaultValue: false, description: 'Call terraform_bin init (needed if modules are added or changes)')
+    ])
+])
+
+node('sumaform-cucumber') {
+    cron('H(0-30) 0-23/4 * * *')
```

Comment:
Ref hosts might be better to be scheduled only nightly

Reply:
Ah, crap. Nice catch. Not "might be better", but "must be". This is a typo.

Reply:
Fixed.